sites/geohub/src/components/Settings.svelte

```diff
@@ -1,41 +1,32 @@
 <script lang="ts">
-  import Notification from '$components/controls/Notification.svelte'
+  import { enhance } from '$app/forms'
+  import { DefaultUserConfig } from '$lib/config/DefaultUserConfig'
+  import {
+    DatasetSortingColumns,
+    MapSortingColumns,
+    NumberOfClassesMaximum,
+    NumberOfClassesMinimum,
+    LimitOptions,
+  } from '$lib/config/AppConfig'
+  import { toast } from '@zerodevx/svelte-toast'
+  import { page } from '$app/stores'
 
-  export let headerHeight: number
+  let userSettings = $page.data.settings ? $page.data.settings : DefaultUserConfig
```

Comment:
Can you move this to get function in `/api/settings/+server.ts`? 

you can use Object.assign to merge objects.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Object.assign(DefaultUserConfig, UserConfig)